proxy_docker/app/script/watchrequest.sh

```diff
@@ -326,7 +326,7 @@ watchtxidrequest() {
   \"event\":\"watchtxid\",
   \"inserted\":\"${inserted}\",
   \"txid\":\"${txid}\",
-  \"confirmedCallbackURL\":${cb1conf_url,
+  \"confirmedCallbackURL\":${cb1conf_url},
```

Comment:
omg *facepalm* thanks

Reply:
:+1 I do this kind of stuff all the time :100: